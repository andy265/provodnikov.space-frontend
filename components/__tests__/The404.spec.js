const { createRenderer } = require('vue-server-renderer')
const { shallow } = require('vue-test-utils')
const The404 = require('~/components/The404')

describe('The404.vue', () => {
  it('has same HTML structure', () => {
    let Component = Object.assign({}, The404, { components: {} })
    let wrapper = shallow(Component)
    let renderer = createRenderer()
    // eslint-disable-next-line
    renderer.renderToString(wrapper.vm, (err, str) => {
      expect(str).toMatchSnapshot()
    })
  })
})
