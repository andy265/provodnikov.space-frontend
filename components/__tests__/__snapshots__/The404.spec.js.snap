// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The404.vue has same HTML structure 1`] = `
<div class="the-404">
    <AppTitle tag="h2" theme="404">404</AppTitle>
    <AppLink to="/">На главную</AppLink>
</div>
`;
